test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -63,6 +63,36 @@ uint64_t clz(uint64_t _v)
 	return r;
 }
 
+/// Count trailing zeros for uint64
+uint64_t ctz(uint64_t _v)
+{
+	uint64_t r = 0;
+	unsigned count = 64;
+	while (count-- && (_v & 1))
```

Comment:
Are we counting zeros or ones? :)

```suggestion
	while (count-- && !(_v & 1))
```